src/containers/cards.jsx

```diff
@@ -19,6 +19,7 @@ import {
 
 import CardsComponent from '../components/cards/cards.jsx';
 import {loadImageData} from '../lib/libraries/decks/translate-image.js';
+import {notScratchDesktop} from '../lib/isScratchDesktop';
```

Comment:
Not for this PR, but do you think this should change from indicating Scratch Desktop specifically to some generic `isThis[Not]ABuildWhichShouldSupportRunningWithoutInternetAccess`? If so, any thoughts on a name that's fewer than 60 characters?

Reply:
`noNeedInternet` maybe? or `shouldHaveInternet` for the reverse?